### DISCUSSION TITLE
The soil surface and potential groundwater table in the 3D model

### DISCUSSION THREAD

**Discussion by qizih** [Anura3D/Anura3D_OpenSource#119] [General]
Dear ayerrocolom and developers,
I am a PHD student, trying to simulate a rainfall indused landslide now.When I try to set the potential groundwater table in the 3D model, I did not find setting options for the 3D model. Similarly, the soil surface settings still retain the 2D "line" boundary settings, and I am unable to set a soil surface in 3D as a plane. May I ask how to solve this problem or do you have any alternative solutions to achieve the setting of soil surface and potential groundwater level in the 3D model?
Looking forwarde to your reply.

**Comment by liseurtong** [comment]
Hi，nice to see your question, are you Timon, i am Pumbaa

Can anyone help answer the same question?

When you're unhappy, please sing with me: Hakuna Matata

**Comment by FrancescaCeccato** [comment]
I'm sorry but this condition is not yet implemented for 3D. You can generate the stress state with gravity loading, i.e. apply only gravity and activate the quasi static convergence.
